src/types/rabbitmq-schemas.ts

```
@@ -0,0 +1,151 @@
+import { z } from 'zod';
```

ESLint

[error] Replace `'zod'` with `"zod"`

(prettier/prettier)